src/vs/base/browser/ui/list/listWidget.ts

```diff
@@ -428,6 +429,14 @@ class TypeLabelController<T> implements IDisposable {
 			if (typeof labelStr === 'undefined' || matchesPrefix(word, labelStr)) {
 				this.list.setFocus([index]);
 				this.list.reveal(index);
+
+				if (index === start) {
```

Comment:
Actually `start` will be `0` when no element is focused, so we probably shouldn't announce it in this case.